src/picongpu/include/simulation_defines/unitless/speciesAttributes.unitless

```diff
@@ -64,6 +66,26 @@ struct UnitDimension<position<T_Type> >
         return unitDimension;
     }
 };
+template<typename T_Type>
+struct MacroWeighted<position<T_Type> >
+{
+    /* the position is identical and can not be scaled by weightings
+     */
+    static bool get()
+    {
+        return false;
+    }
+};
+template<typename T_Type>
+struct WeightingPower<position<T_Type> >
+{
+    /* x * weighting^0 == x: same for real and macro particle
```

Comment:
Could you change one line comments to one line?

`/* comment */` or `// comment`